src/overlays/actors/ovl_Obj_Lightblock/z_obj_lightblock.h

```diff
@@ -3,15 +3,20 @@
 
 #include "global.h"
 
+#define LIGHTBLOCK_TYPE(thisx) ((thisx)->params & 1)
+#define LIGHTBLOCK_DESTROYED(thisx) (((thisx)->params >> 8) & 0x7F)
+
 struct ObjLightblock;
 
 typedef void (*ObjLightblockActionFunc)(struct ObjLightblock*, GlobalContext*);
 
 typedef struct ObjLightblock {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x64];
+    /* 0x0000 */ DynaPolyActor dyna;
+    /* 0x015C */ ColliderCylinder collider;
     /* 0x01A8 */ ObjLightblockActionFunc actionFunc;
-    /* 0x01AC */ char unk_1AC[0x4];
+    /* 0x01AC */ s8 timer;
+    /* 0x01AD */ u8 opacity;
```

Comment:
Per something Fig mentioned to me in my En_Holl PR, "opacity" is typically referred to as "alpha" in a programming context, and specifically in the standard libraries used for this game. It's the "alpha" component of RGBA (not to be confused with "gamma", which adjusting can help you see in dark games better- that tripped me up for a while).
```suggestion
    /* 0x01AD */ u8 alpha;
```